rfps/rfp-filecoin-wallets.md

```diff
@@ -0,0 +1,159 @@
+# RFP: Filecoin Wallet & Tools
+
+**Name of Project:** `Filecoin Wallet & Tools`
+
+**Proposal Category:** `app-dev`, `devtools-libraries`
+
+## Project Description
+
+This RFP has 2 possible tracks:
+
+  - [Track 1: Self-hosted Wallet](#track-1-self-hosted-wallet)
+  - [Track 2: Wallet Tools and Integrations](#track-2-wallet-tools-and-integrations)
+  
+### Track 1: Self-hosted Wallet
+--------------------------------
+We are seeking proposals for an open source, self-hosted Filecoin wallet application that supports receiving, storing and sending FIL, shows when transactions are confirmed by the network and helps users safely manage their keys.
+
+> NOTE: Nodes have wallet CLIs and APIs that will be further developed to meet the Filecoin Spec (e.g. Multisig wallets). A reference tool for signing raw transactions outside of a node will also be created by the Filecoin team.
```

Comment:
If we say this, we should back it up with timing. Do you have this info?

Reply:
No we don't have explicit timings. One is a filed Issue in Github.

Reply:
Can we add something after the last sentence here that says "We don't yet have exact timing for this functionality, but you can assume their existence if that is helpful to you in development."

Reply:
Kubuxu and Magik6k are now scoping this so I assume this will be soon now.